developing/languages/BASH.md

```diff
@@ -37,6 +38,17 @@ Shell programming is deceptively perilous (see [Bash Pitfalls](https://mywiki.wo
 * [Don't](https://mywiki.wooledge.org/DontReadLinesWithFor) iterate through lines in a stream with `for`.
 * Prefer `[[` to `[`: `[[` is a bash extension that is a strict upgrade over `[`.
 
+## PATH
+
+* `PATH` is a shortcut to a certain location on your machine, which you can view by typing `echo $PATH` in the terminal. To oversimplify, a `PATH` is an environment variable which consists of a list of directories. Adding a `PATH` to your `bash` profile will give your machine direct instructions on where to look for a certain program.
+* One of several files can affect how your machine searches for a given `PATH`, listed in the order a Mac searches for them:
+
+  * `~/.bashrc` (read on every new shell, so looked at far more often)
+  * `~/.bash_profile` (read only on login)
+  * `~/.profile`
+
+* If you have multiple files affecting your `PATH`, consider consolidating them into one `/.bashrc` and deleting the others. Next, organize each `PATH` directory in your `~/.bashrc` from most-used to least-used to maximize readability.
```

Comment:
The reason for ordering by most-used is for efficiency, not readability.

Reply:
As stated above. Good catch.